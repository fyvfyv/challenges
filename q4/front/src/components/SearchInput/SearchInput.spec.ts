import { mount } from '@vue/test-utils'
import Component from './index'

describe('SearchInput', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Component, {
      propsData: {
        value: ''
      }
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders correctly', () => {
    const wrapper = mount(Component, {
      propsData: {
        value: ''
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
