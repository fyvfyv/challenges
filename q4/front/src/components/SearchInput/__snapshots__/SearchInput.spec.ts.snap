// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput renders correctly 1`] = `
<div
  class="wrapper"
>
  <label
    for="searchField"
  >
    Search:
  </label>
   
  <input
    id="searchField"
  />
</div>
`;
